tests/snapshots/assets__check_file@packages-fletcher-draw.typ-120.snap

```diff
@@ -123,12 +123,15 @@ snapshot_kind: text
   let θ-normal = θ + if edge.label-side == right { +90deg } else { -90deg }
 
   if type(edge.label-angle) == alignment {
-    edge.label-angle = θ - (
-      right: 0deg,
-      top: 90deg,
-      left: 180deg,
-      bottom: 270deg,
-    ).at(repr(edge.label-angle))
+    edge.label-angle = (
+      θ
+        - (
+          right: 0deg,
+          top: 90deg,
+          left: 180deg,
+          bottom: 270deg,
+        ).at(repr(edge.label-angle))
+    )
   } else if edge.label-angle == auto {
```

Comment:
idk what this is changed. This testcase is 120 wide

Reply:
it may look better without consideration of flavor